### PR TITLE
Various fixes needed for IE11 compatibility

### PR DESCRIPTION
**Pull request by vrockai**

### CODE REVIEW

**Comment by vrockai** [comment]
Hey @chmanie , please let me now if you won't (for any reason) merge this, so I can start working on a local workaround. Thanks!

File: lib/interceptor.js [hunk 1]
```diff
@@ -50,7 +50,6 @@ var interceptor = {
         }
         if (init) {
           request.requestBody = init.body;
-          request.url = init.url;
```
Comment (chmanie) [inline lib/interceptor.js]: Why did you remove this?

Comment (chmanie) [reply lib/interceptor.js]: Oops, sorry, just saw your issue. Never mind then

Comment (chmanie) [reply lib/interceptor.js]: Are the changes made by @lacell75 acceptable for you here: https://github.com/chmanie/wdio-intercept-service/pull/56/files?

**Comment by chmanie** [comment]
Thank you very much for your PR. I highly appreciate it! Sorry I couldn't get to it earlier.

**Comment by chmanie** [comment]
Published v3.0.3 which incorporates this fix. Thanks again!

File: lib/interceptor.js (hunk 1 above)
Comment (vrockai) [reply lib/interceptor.js]: Since `init.url` is an invalid option (see https://github.com/chmanie/wdio-intercept-service/issues/51) I don't see the need to use it.

**Comment by vrockai** [comment]
Thanks a lot, we had a workaround using my personal fork, but we can revert back to use your awesome package now!

**Comment by lacell75** [comment]
with my fix it should work on IE 11  :wink:

**Comment by vrockai** [comment]
@lacell75 we're already using fixes from this PR with IE11. IE11 should work just fine with this merged. Do you still have issues?

**Comment by lacell75** [comment]
Now, I'm using the 3.0.3 published and I don't have issues. That works!

**Comment by chmanie** [comment]
Happy to see that it works for you!
